### PR TITLE
alerts-ui: Update navbar items

### PR DESCRIPTION
**Pull request by mozky**
Problem

This small PR fixes some alignment issues on the navbar and the language selector button.

Solution

Modifications to the Navbar's HTML file to fixe Bootstrap's navbar.

Result

A nicer navbar!

### CODE REVIEW

**Comment by AlexITC** [comment]
Hey @mozky 

I have tried the changes locally and the navbar items disappear on small screens, this will stop users from registering using mobile devices, see:
navbar

Please fix that in order to integrate your changes.

Thanks for your work.

**Comment by mozky** [comment]
@AlexITC Pushed a fix for mobile view.

File: alerts-ui/src/app/home/home.component.css [hunk 1]
```diff
@@ -15,4 +14,12 @@
...
   font-size: 20px;
-}
\ No newline at end of file
+}
+
+.max-width {
+  max-width: 100%;
+}
+
+.container {
```
Comment (AlexITC) [inline alerts-ui/src/app/home/home.component.css]: I'm not sure on updating the `container`, can we use a custom class instead?

File: alerts-ui/src/app/app-navbar/app-navbar.component.html [hunk 2]
```diff
@@ -1,6 +1,13 @@
-<nav class="navbar navbar-default">
+<nav class="navbar navbar-default navbar-fixed-top">
   <div class="container-fluid">
     <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse" [attr.aria-expanded]="!collapsed"
+        (click)="toggleCollapse()" [ngClass]="{'collapsed': collapsed}">
+        <span class="sr-only">Toggle navigation</span>
```
Comment (AlexITC) [inline alerts-ui/src/app/app-navbar/app-navbar.component.html]: is this text displayable? if it is, we need to use i18n, the approach would be like `{{{'label.toggleNavigation' | translate}}}`

**Comment by AlexITC** [comment]
@mozky the new view looks good, just a couple of comments.

Thanks.

**Comment by mozky** [comment]
@AlexITC Added sticky footer and some other minor updates

File: alerts-ui/src/app/app-navbar/app-navbar.component.html [hunk 3]
```diff
@@ -49,10 +49,10 @@
               </button>
               <!-- TODO: load langs dynamically -->
               <ul *dropdownMenu class="dropdown-menu">
-                <li>
+                <li [ngClass] = "{'disabled': languageService.getLang() === 'en'}">
```
Comment (AlexITC) [inline alerts-ui/src/app/app-navbar/app-navbar.component.html]: great!

**Comment by AlexITC** [comment]
As a suggestion, try to keep commits a bit clean, let's say, if the message says `update the navbar`, it's better to not touch something else and use a different commit instead, besides that, well done, thanks!

**Pull request merged**